paasta_tools/delete_kubernetes_deployments.py

```diff
@@ -73,7 +73,10 @@ def main() -> None:
     for deployment_name in deployment_names:
         try:
             log.debug(f"Deleting {deployment_name}")
-            delete_deployment(kube_client=kube_client, deployment_name=deployment_name)
+            delete_deployment(
```

Comment:
should we undo this change to make the diff smaller? doesn't look like we actually change the code in this file

Reply:
yep, I think this one was changed by the pre-commit hooks when I tried to add an argument then deleted it. Will fix it